### ISSUE TITLE
Feature request: Ignore

### ISSUE THREAD

**Issue by joshlopes** [bdd/runitor#38]
This is a workaround for a feature that was already requested by someone to healthchecker https://github.com/healthchecks/healthchecks/issues/525 . There was some people that said they are dealing with it on a much lower level when running the command/script so I was thinking runitor could also do that.

## Suggestion

Add `-fault-tolerance` which would look like `-fault-tolerance 2` which in summary the command that was being ran failed (exited code > 0) then it would catch the error and output and send as a "SUCCESS" to healthchecker, it was still a failure but flagged as a success. 

## How should this work

Basically runitor would have to look for "fault-tolerance" option and create a file say `fault-tolerance-uuid.txt` with the number of attempts it already did, so if it fails it creates/modifies the file to whatever content + 1, and if that content + 1 > fault tolerance it would fail otherwise it sends an OK HTTP with the actual buggy output for debug purposes. If it passed the file should be removed.
